src/Symfony/Component/Validator/Constraints/AtLeastOne.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+/**
+ * @Annotation
+ * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
+ *
+ * @author Przemysław Bogusz <przemyslaw.bogusz@tubotax.pl>
+ */
+class AtLeastOne extends Composite
```

Comment:
`AtLeastOneOf` ? (would it even make sense to simplify to `Or` ?)

Reply:
Isn't `Or` reserved keyword?

Reply:
done